workspaces/rbac/plugins/rbac/src/components/CreateRole/RoleDetailsForm.tsx

```diff
@@ -69,6 +71,19 @@ export const RoleDetailsForm = ({
         onBlur={handleBlur}
         multiline
       />
+      <TextField
+        label="Owner"
+        variant="outlined"
+        helperText="Optional: Choose a user or group who will have permission to edit this role and create additional roles. In the next step, specify which users they can assign to their roles and which plugins they can grant access to."
```

Comment:
Since this is just a text-field instread of choose shouldnt this be enter or add a user/group?
```suggestion
        helperText="Optional: Enter a user or group who will have permission to edit this role and create additional roles. In the next step, specify which users they can assign to their roles and which plugins they can grant access to."
```